### ISSUE TITLE
LevelDB issue: "Could not locate LevelDOWN"

### ISSUE THREAD

**Issue by cardoni** [jcblw/pastila#40]
# Issue

Upon running `npm start`, app throws the following error:

```
Uncaught Exception:
LevelUPError: Could not locate LevelDOWN, try `npm install leveldown`
    at getLevelDOWN (/Users/greg/development/other/pastila/app/node_modules/levelup/lib/util.js:71:11)
    at LevelUP.open (/Users/greg/development/other/pastila/app/node_modules/levelup/lib/levelup.js:113:37)
    at new LevelUP (/Users/greg/development/other/pastila/app/node_modules/levelup/lib/levelup.js:86:8)
    at LevelUP (/Users/greg/development/other/pastila/app/node_modules/levelup/lib/levelup.js:46:12)
    at Object.<anonymous> (/Users/greg/development/other/pastila/app/index.js:20:8)
    at Module._compile (module.js:418:26)
    at Object.Module._extensions..js (module.js:436:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:298:12)
    at Object.<anonymous> (/Users/greg/development/other/pastila/node_modules/electron-prebuilt/dist/Electron.app/Contents/Resources/default_app/main.js:49:23)
```

> **Note**
> Same error occurs upon opening `Pastila.app`, after running `npm run build`.
## Attempted Solution

Installing `leveldown` (locally or globally) doesn't appear to resolve this issue.
## Screenshot

![Pastila Error](https://cloud.githubusercontent.com/assets/674107/8686930/b896b550-2a3f-11e5-8a75-e1ec31550615.png)
## Relevant System Details

**OS**: Mac OS X version `10.11 Beta (15A216g)`
**Node**: version `0.12.7`

**Comment by jcblw** [comment]
you will need iojs... even tho it seems that you didnt install anything. Please run `git clean -xdf` in the dir first then install while using iojs. Preferably >1.6 Then let me know :)

**Comment by cardoni** [comment]
Snap. Doesn't work with iojs either:
https://gist.github.com/cardoni/f542e18f4b31b95cb489

Looking at my gist, I think there's a chance this is related to my running `OS X 'El Capitan' Beta 3`.
You'll notice a few warnings (although no errors) during the compilation step for LevelDB.

I'll do a little more digging to see if I can find anyone else with leveldb/leveldown-specific related issues with the latest mac os x beta.  I'd love to find a fix so I can play with your app!

**Comment by jcblw** [comment]
SHIT :hankey: Ill take a look at this. I was getting a similiar issue on my Ubuntu box :/

**Comment by jcblw** [comment]
soo I was able to get things running with 1.2.0 of iojs. I have to look at the current version that electron is using because the native installs require it to be about the same version for them to work properly... Ill keep this open to find that version and add it to the repo readme!

**Comment by jcblw** [comment]
looking into this further but it looks like right now the fix is to use 1.2.0. I added it to the readme I think im going to implement something that is less dependent on the node version your running.
